h2o-py/tests/testdir_parser/pyunit_PUBDEV_5705_drop_columns_parser_svmlight_fail.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import print_function
+import sys
+import traceback
+
+sys.path.insert(1, "../../")
+import h2o
+from tests import pyunit_utils
+import os
+from h2o.exceptions import H2OResponseError
+
+
+def test_parser_svmlight_column_skip_not_supported():
+    print("Test that functions calling fail if skipped_columns is passed with svm file.")
+    # generate a big frame with all datatypes and save it to svmlight
+    nrow = 10
+    ncol = 10
+    seed = 12345
+
+    f1 = h2o.create_frame(rows=nrow, cols=ncol, real_fraction=0.5, integer_fraction=0.5, missing_fraction=0.2,
+                          has_response=False, seed=seed)
+
+    tmpdir = os.path.normpath(os.path.join(os.path.dirname(os.path.realpath('__file__')), "..", "results"))
+    if not (os.path.isdir(tmpdir)):
+        os.mkdir(tmpdir)
+    savefilenamewithpath = os.path.join(tmpdir, 'out.svm')
+    pyunit_utils.write_H2OFrame_2_SVMLight(savefilenamewithpath, f1)  # write h2o frame to svm format
+
+    try:
+        print("Test upload SVM file. "
+              "Expected result is Java exception error: skipped_columns not supported for AVRO and SVMlight")
+        h2o.upload_file(savefilenamewithpath, skipped_columns=[5])
+        assert False, "Test should have thrown an exception due skipped_columns parameter is present"  # should have failed here
+    except H2OResponseError as e:
+        print("Test OK, finished with H2OResponseError")
+        assert "skipped_columns are not supported" in str(e.args[0].exception_msg), "Exception message is different"
+
+    try:
+        print("Test import SVM file. "
+              "Expected result is Java exception error: skipped_columns not supported for AVRO and SVMlight")
+        h2o.import_file(savefilenamewithpath, skipped_columns=[5])
+        assert False, "Test should have thrown an exception due skipped_columns parameter is present"  # should have failed here
+    except H2OResponseError as e:
+        print("Test OK, finished with H2OResponseError")
+        assert "skipped_columns are not supported" in e.args[0].exception_msg
```

Comment:
Generally prints in tests are not good practice, usually they should be just asserts, prints are pseudo-asserts for the programmer reading the test. I know its common practice in our py/R tests, but just wanted to point out that there are better way to do things.
Also you might want to switch the order of the `print` and `assert` because it will be confusing when the test fails with printout `Test OK` and `assertion error` afterwards.

Reply:
Order changed and I will keep in mind to prefer asserts before print. Thanks